### PR TITLE
docs: add form to estimate realtime throughput

### PR DESCRIPTION
**Pull request by egor-romanov**
## What kind of change does this PR introduce?

- add a new component with a form to estimate realtime possible throughput in the docs

### CODE REVIEW

File: apps/docs/components/RealtimeLimitsEstimater.tsx [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+            {table
+              .filter(
+                (l) => l.computeAddOn === computeAddOn && l.filters === filters && l.rls === rls
+              )
+              .map((l) => (
+                <Select.Option
+                  value={l.concurrency.toString()}
+                  selected={l.concurrency === concurrency}
+                >
+                  {l.concurrency.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
```
Comment (charislam) [inline apps/docs/components/RealtimeLimitsEstimater.tsx:347]: maybe `Intl.NumberFormat`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format

File: apps/docs/components/RealtimeLimitsEstimater.tsx [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+            <Select.Option value="small">Small to medium</Select.Option>
+            <Select.Option value="large">Large to 16XL</Select.Option>
+          </Select>
+        </div>
+
+        <div>
+          <label htmlFor="filters">Filters:</label>
+          <Select
+            id="filters"
+            style={{ fontFamily: 'monospace' }}
```
Comment (charislam) [inline apps/docs/components/RealtimeLimitsEstimater.tsx:314]: see above about classname

File: apps/docs/components/RealtimeLimitsEstimater.tsx [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+            onChange={handleFiltersSelection}
+          >
+            <Select.Option value="false">No</Select.Option>
+            <Select.Option value="true">Yes</Select.Option>
+          </Select>
+        </div>
+
+        <div>
+          <label htmlFor="rls">RLS:</label>
+          <Select id="rls" style={{ fontFamily: 'monospace' }} onChange={handleRLSSelection}>
```
Comment (charislam) [inline apps/docs/components/RealtimeLimitsEstimater.tsx:325]: see above about classname

File: apps/docs/components/RealtimeLimitsEstimater.tsx [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+            <Select.Option value="false">No</Select.Option>
+            <Select.Option value="true">Yes</Select.Option>
+          </Select>
+        </div>
+
+        <div>
+          <label htmlFor="concurrency">Concurrency:</label>
+          <Select
+            id="concurrency"
+            style={{ fontFamily: 'monospace' }}
```
Comment (charislam) [inline apps/docs/components/RealtimeLimitsEstimater.tsx:335]: see above about classname

File: apps/docs/components/RealtimeLimitsEstimater.tsx [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+                <th className="px-4 py-2">Max total messages /sec</th>
+                <th className="px-4 py-2">Latency p95</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td className="border px-4 py-2">{limits.maxDBChanges}</td>
+                <td className="border px-4 py-2">{limits.maxMessagesPerClient}</td>
+                <td className="border px-4 py-2">
+                  {limits.totalMessagesPerSecond.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
```
Comment (charislam) [inline apps/docs/components/RealtimeLimitsEstimater.tsx:372]: see above about number format

File: apps/docs/pages/guides/realtime/postgres-changes.mdx [hunk 6]
```diff
@@ -1361,45 +1362,13 @@ Realtime systems usually require forethought because of their scaling dynamics.
 
 There can be a database bottleneck which limits message throughput. If your database cannot authorize the changes rapidly enough, the changes will be delayed until you receive a timeout.
 
+Please note that currently to maintain the order of changes, we are using a single thread to process changes on the database side. This means that compute upgrades will not improve the performance of this feature by much, refer to form below to estimate the maximum throughput you can expect from your database instance.
```
Comment (charislam) [inline apps/docs/pages/guides/realtime/postgres-changes.mdx:1365]:
```suggestion
Database changes are processed on a single thread to maintain the change order. That means compute upgrades don't have a large effect on the performance of Postgres change subscriptions. You can estimate the expected maximum throughput for your database below.
```

File: apps/docs/pages/guides/realtime/postgres-changes.mdx [hunk 7]
```diff
@@ -1361,45 +1362,13 @@ Realtime systems usually require forethought because of their scaling dynamics.
...
-| 🚫      | 🚫        | 5,000              | 10                            | 10                            | 50,000                      | 672              |
-| 🚫      | 🚫        | 10,000             | 5                             | 5                             | 50,000                      | 1253             |
-| 🚫      | 🚫        | 35,000             | 1                             | 1                             | 35,000                      | 1257             |
-| 🚫      | ✅        | 500                | 40                            | 8                             | 4,000                       | 618              |
-| 🚫      | ✅        | 2,000              | 10                            | 2                             | 4,000                       | 606              |
-| 🚫      | ✅        | 4,000              | 5                             | 1                             | 4,000                       | 918              |
+You can estimate what maximum throughput you can expect from your database instance using the following form:
```
Comment (charislam) [inline apps/docs/pages/guides/realtime/postgres-changes.mdx:1369]:
```suggestion
Enter your database settings to estimate the maximum throughput for your instance:
```

**Review by charislam — APPROVED** [review]

**Comment by egor-romanov** [comment]
@charislam thank you very much!
If you don't mind, take another quick look please, cause i also returned a table view hidden by collapsible. It is a bit not smooth, but this is a component problem, I checked in studio, it works the same way there

**Pull request merged**